examples/tree_1d_dgsem/elixir_traffic_flow_lwr_trafficjam.jl

```diff
@@ -0,0 +1,80 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+
+equations = TrafficFlowLWREquations1D()
+
+# Use first order finite volume to prevent oscillations at the shock
+solver = DGSEM(polydeg = 0, surface_flux = flux_lax_friedrichs)
```

Comment:
Is this selection of `polydeg=0` just for academic purposes? So this (presumably) uses the Legendre-Gauss nodes such that it is the cell center. This is a somewhat roundabout way to demonstrate such functionality.

Reply:
Actually only for constant polynomials I could suppress oscillations at the discontinuity, even when using `ShockcapuringHennemannGassner`.

Reply:
Right, the shock capturing from Hennemann is not monotonicity preserving.